lib/orbit/import/import_row_certifications.ex

```diff
@@ -0,0 +1,8 @@
+defmodule Orbit.Import.ImportRoWCertifications do
+  @behaviour Orbit.Import.Importer
+
+  @impl true
+  def import!(file_lines) do
+    Orbit.Import.ImportCertifications.import!(file_lines, :right_of_way)
```

Comment:
This needs a whole extra file and layer of indirection in order to pass the `:right_of_way` argument into `ImportCertifications`. Is there a way we could remove that, maybe by having the `import!()` callback take an optional argument (specified in the crontab arguments) so that `ImportCertifications` can be the module that implements the behaviour?